ocrd/ocrd/decorators/ocrd_cli_options.py

```diff
@@ -33,6 +34,8 @@ def cli(mets_url):
         parameter_option,
         parameter_override_option,
         loglevel_option,
+        option('--agent_type', type=click.STRING),
```

Comment:
I like the suggestion. This is much cleaner, yes. The only difference is that I will still stick to `agent_type` instead of `type_` internally. Not a big fan of hacky workarounds that are hard to read and follow than more descriptive naming conventions. The core base is already hard to follow (for external developers) due to the same reason... In case you missed it, check the discussion [here](https://github.com/OCR-D/core/pull/974#discussion_r1145327980) as well.